change_level.py

```diff
@@ -29,7 +29,11 @@
     if not 0 <= int(level) <= 10:
         print "Error: app level must be between 0 and 10, it's '%s'" % level
         sys.exit(1)
-
-    app_list[app_id]["level"] = int(level)
+        
+    if app_list[app_id]["state"] == "working":
+        app_list[app_id]["level"] = int(level)
+    else:
```

Comment:
Hmm I think that might trigger an issue if the level key does not exist :P

```suggestion
    elif "level" in app_list[app_id]:
```